### PR TITLE
Create translation-guide.md

### PR DESCRIPTION
**Pull request by hanyujie2002**
The translation guide is based on that of MDN https://github.com/mdn/translated-content/blob/main/docs/zh-cn/translation-guide.md

Relates to https://github.com/ts-zh-docs/community/issues/5

### CODE REVIEW

File: docs/translation-guide.md [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+> 若你对本文档使用的 Markdown 格式不熟悉，请参阅[如何使用 markdown 来撰写文档][]。
+
+## 元数据
+
+每个文档的开头都会有元数据，本地化文档中应该使用的元数据说明如下：
+
+- `title`（必须）：为文档的大标题
+- `layout`（必须）
+- `permalink`（必须）：为与网页 URL 相关的元数据
+
```
Comment (hanyujie2002) [inline docs/translation-guide.md]:
```suggestion
- `permalink`（必须）：为与网页 URL 相关的元数据
- `oneline`（必须）：用一句话简要介绍文章内容
```

File: docs/translation-guide.md [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+如果你对本文的指南有任何改进的建议，请提出问题（issue）或发起 PR。我们对此表示欢迎。下面进入简体中文翻译指南的正文部分。
+
+> 若你对本文档使用的 Markdown 格式不熟悉，请参阅[如何使用 markdown 来撰写文档][]。
+
+## 元数据
+
+每个文档的开头都会有元数据，本地化文档中应该使用的元数据说明如下：
+
+- `title`（必须）：为文档的大标题
+- `layout`（必须）
```
Comment (hanyujie2002) [inline docs/translation-guide.md]:
```suggestion
- `layout`（必须）：目前统一为 `docs`
```

File: docs/translation-guide.md [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+- `permalink`（必须）：为与网页 URL 相关的元数据
+
+最终在简体中文文档中呈现的元数据如下所示：
+
+```yaml
+// todo
```
Comment (hanyujie2002) [inline docs/translation-guide.md]:
```suggestion
title: 标题
layout: docs
permalink: /docs/handbook/2/example.html
oneline: "对文章内容的简要介绍"
```

File: docs/translation-guide.md [hunk 4]
```diff
@@ -0,0 +1,190 @@
...
+对于站内链接，请只保留 URL 中的路径及其之后的部分（去除协议和域名）。
+
+例如：
+
+```md
+If you are new to HTML, CSS, JavaScript, etc., try our [Learn web development](/docs/Learn) tutorials.
+```
+
+在翻译时，应该将 URL 路径开头的 `/en-US` 替换为 `/zh-CN`：
+
```
Comment (hanyujie2002) [inline docs/translation-guide.md]:
```suggestion
```

File: docs/translation-guide.md [hunk 5]
```diff
@@ -0,0 +1,190 @@
...
+```md
+如果你刚刚开始了解 HTML、CSS、JavaScript 等相关知识，你可以试着完成我们的 [Web 开发教程](/docs/Learn)。
+```
```
Comment (hanyujie2002) [inline docs/translation-guide.md]:
```suggestion
```

File: docs/translation-guide.md [hunk 6]
```diff
@@ -0,0 +1,188 @@
...
+> 若你对本文档使用的 Markdown 格式不熟悉，请参阅[如何使用 markdown 来撰写文档][]。
+
+## 元数据
+
+每个文档的开头都会有元数据，本地化文档中应该使用的元数据说明如下：
+
+- `title`（必须）：为文档的大标题
+- `layout`（必须）：目前统一为 `docs`
+- `permalink`（必须）：为与网页 URL 相关的元数据
+- `oneline`（必须）：用一句话简要介绍文章内容
```
Comment (hanyujie2002) [inline docs/translation-guide.md:16]:
```suggestion
- `oneline`（必须）：用一句话简要介绍文章内容

```

File: docs/translation-guide.md [hunk 7]
```diff
@@ -0,0 +1,189 @@
...
+## 站内链接翻译
+
+对于站内链接，请只保留 URL 中的路径及其之后的部分（去除协议和域名）。
+
+例如：
+
+```md
+If you are new to HTML, CSS, JavaScript, etc., try our [Learn web development](/docs/Learn) tutorials.
+```
+
```
Comment (hanyujie2002) [inline docs/translation-guide.md:54]:
```suggestion
```

File: docs/translation-guide.md [hunk 8]
```diff
@@ -0,0 +1,188 @@
...
+这是一个示例。我们不应该断行写这一段话。就像这样。
+```
+
+### 词语翻译
+
+### 代名词
+
+<!-- markdownlint-disable search-replace -->
+
+我们无需将“you”翻译为“您”，在文档正文部分的翻译中，请统一使用“你”。
```
Comment (awxiaoxian2020) [inline docs/translation-guide.md]: My preference.
cc: @hanyujie2002 @Jeffzholy @EnochGao 
```suggestion
我们无需将“you”翻译为“您”，在文档正文部分的翻译中，请统一使用“你”。且在中文技术文档中，一般语气风格较为客观，在交互性不是很强的情况下，可以考虑删去“你”这样的主语，显得更符合中文表达习惯。
```

File: docs/translation-guide.md [hunk 9]
```diff
@@ -0,0 +1,188 @@
...
+
+我们在仓库中维护了一个[术语表](glossary.md)，旨在统一目前还未有明确中文翻译的术语。在翻译文档时，请尽量使用术语表中的术语，以保证文档的一致性。
+
+> 如果术语表中缺少某些术语，或者你对其中的术语翻译有任何疑问，欢迎提出[议题](https://github.com/ts-zh-docs/TypeScript-zh-Website/issues/new)或发起 PR。
+
+### 常用标题
+
+英文文档中使用了一些常见标题，为了保持简体中文文档的一致性，请在翻译时参考术语表中的[标题和表格的翻译](glossary.md#标题表格)。
+
+[如何使用 markdown 来撰写文档]: https://developer.mozilla.org/zh-CN/docs/MDN/Writing_guidelines/Howto/Markdown_in_MDN
```
Comment (awxiaoxian2020) [inline docs/translation-guide.md]: 187 188 不建议保留

**Review by awxiaoxian2020 — CHANGES_REQUESTED** [review]

File: docs/translation-guide.md [hunk 10]
```diff
@@ -0,0 +1,188 @@
...
+[运行实例]: https://developer.mozilla.org/zh-CN/docs/MDN/Writing_guidelines/Page_structures/Live_samples
```
Comment (hanyujie2002) [inline docs/translation-guide.md]:
```suggestion
```

File: docs/translation-guide.md [hunk 11]
```diff
@@ -0,0 +1,186 @@
...
+### 术语表
+
+我们在仓库中维护了一个[术语表](glossary.md)，旨在统一目前还未有明确中文翻译的术语。在翻译文档时，请尽量使用术语表中的术语，以保证文档的一致性。
+
+> 如果术语表中缺少某些术语，或者你对其中的术语翻译有任何疑问，欢迎提出[议题](https://github.com/ts-zh-docs/TypeScript-zh-Website/issues/new)或发起 PR。
+
+### 常用标题
+
+英文文档中使用了一些常见标题，为了保持简体中文文档的一致性，请在翻译时参考术语表中的[标题和表格的翻译](glossary.md#标题表格)。
+
```
Comment (hanyujie2002) [inline docs/translation-guide.md]:
```suggestion
```

File: docs/translation-guide.md [hunk 12]
```diff
@@ -0,0 +1,185 @@
+# 针对 TypeScript 文档的本地化指南
+
+本文是针对简体中文（zh-CN）文档的翻译指南。
+
+如果你对本文的指南有任何改进的建议，请提出问题（issue）或发起 PR。我们对此表示欢迎。下面进入简体中文翻译指南的正文部分。
+
+> 若你对本文档使用的 Markdown 格式不熟悉，请参阅[如何使用 markdown 来撰写文档][]。
```
Comment (awxiaoxian2020) [inline docs/translation-guide.md]: here

**Review by awxiaoxian2020 — CHANGES_REQUESTED** [review]

File: docs/translation-guide.md (hunk 12 above)
Comment (hanyujie2002) [inline docs/translation-guide.md]:
```suggestion
> 若你对本文档使用的 Markdown 格式不熟悉，请参阅[如何使用 markdown 来撰写文档][https://developer.mozilla.org/zh-CN/docs/MDN/Writing_guidelines/Howto/Markdown_in_MDN]。
```

**Comment by awxiaoxian2020** [comment]
We should move these docs to `zh` repo. But now, let's merge it.

**Pull request merged**